mlir/utils/jenkins/Jenkinsfile

```diff
@@ -177,16 +177,15 @@ mlir/utils/jenkins/miopen-tests/**\
                 stage("Performance testing") {
                     when {
                         beforeAgent true;
-                        allOf {
+                        {
```

Comment:
In cleanup
1) Does that extra pair of braces give us anything?
2) Is it a syntax error?
3) Could you reindent?

Reply:
Fixed.